packages/cli/src/ui/hooks/useAutoAcceptIndicator.ts

```diff
@@ -34,6 +34,21 @@ export function useAutoAcceptIndicator({
       let nextApprovalMode: ApprovalMode | undefined;
 
       if (key.ctrl && key.name === 'y') {
+        if (
+          config.isYoloModeDisabled() &&
+          config.getApprovalMode() !== ApprovalMode.YOLO
+        ) {
+          if (addItem) {
+            addItem(
+              {
+                type: MessageType.INFO,
```

Comment:
make this message more explicit about not being able to enter yolo mode as it is disabled in your settings.  and make this a warning rather than just info